### PR TITLE
Implement support for timezone in dateTimes from user

### PR DESCRIPTION
**Pull request by bartero**
* ISO8601 is mandatory for some input dateTimes
* ISO8601 will be mandatory for all inpute dateTimes soon
* Requested dates through API are converted to the request timezone
* Requested dates through API are formatted to the ISO8601 format

### CODE REVIEW

File: src/Gzero/Cms/Http/Resources/Content.php [hunk 1]
```diff
@@ -128,9 +128,11 @@ public function toArray($request)
             'is_promoted'        => $this->is_promoted,
             'is_sticky'          => $this->is_sticky,
             'path'               => $this->buildPath($this->path),
-            'published_at'       => ($this->published_at) ? $this->published_at->toIso8601String() : null,
-            'created_at'         => $this->created_at->toIso8601String(),
-            'updated_at'         => $this->updated_at->toIso8601String(),
+            //'published_at'       => ($this->published_at) ? $this->published_at
```
Comment (AdrianSkierniewski) [inline src/Gzero/Cms/Http/Resources/Content.php]: We don't want that comment here, right?

File: routes/api.php [hunk 2]
```diff
@@ -13,7 +13,7 @@
     /** @var \Illuminate\Routing\Router $router */
 
     // ======== Contents ========
-    $router->get('contents', 'ContentController@index');
+    $router->get('contents', 'ContentController@index')->name('contents');
```
Comment (AdrianSkierniewski) [inline routes/api.php]: I see why you needed that name here, but we're not using named routes in api right now, right? We'd need to guarantee that all those names are unique in whole app, so probably better would be to use something like `api.contents`.

BTW What was wrong with this approach `apiUrl("contents?updated_at=$yesterday,$tomorrow"`?

Comment (bartero) [reply routes/api.php]: * I have prefixed the api route names to with `api.`
* I wrote tests checking the output from the `route(...)`

**Comment by bartero** [comment]
* I have prefixed the api route names to with api.
* I wrote tests checking the output from the `route(...)`

File: src/Gzero/Cms/Http/Resources/Content.php (hunk 1 above)
Comment (bartero) [reply src/Gzero/Cms/Http/Resources/Content.php]: I removed it

**Pull request merged**